pkgs/development/tools/rust/ravedude/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, rustPlatform
+, fetchCrate
+, pkg-config
+, udev
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "ravedude";
+  version = "0.1.5";
+
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "wcY9fvfIn1pWMAh5FI/QFl18CV2xjmRGSwwoRfGvujo=";
```

Comment:
```suggestion
    hash = "wcY9fvfIn1pWMAh5FI/QFl18CV2xjmRGSwwoRfGvujo=";
```

Reply:
Good idea, thanks for pointing it out.

I had to prefix the hash with its type, to make it [SRI-compatible](https://github.com/NixOS/rfcs/pull/131)). Otherwise, the build would fail as it cannot infer the algorithm:

```
error: hash 'wcY9fvfIn1pWMAh5FI/QFl18CV2xjmRGSwwoRfGvujo=' does not include a type, nor is the type otherwise known from context
```